src/V3Error.h

```diff
@@ -147,6 +147,8 @@ class V3ErrorCode final {
         VARHIDDEN,      // Hiding variable
         WAITCONST,      // Wait condition is constant
         WIDTH,          // Width mismatch
+        WIDTHTRUNCATE,  // Width mismatch- lhs < rhs
+        WIDTHEXTEND,    // Width mismatch- lhs > rhs
```

Comment:
I'm not sure how they make it clearer, what would the other interpretation be?

Reply:
Also in src/V3Width.cpp There's one warn width still in there, the x/z extend. I wanted to leave that as a normal width because I think that's worse than a zero extend. Do you think I should switch that one as well?

Reply:
I could rename them to match the slang expand/trunc warnings https://sv-lang.com/warning-ref.html#width-trunc

Reply:
I like matching slang.  Perhaps also make the XZ one into "WIDTHXZ"?